### ISSUE TITLE
A question about LUA_PACKAGE_PATH

### ISSUE THREAD

**Issue by idevz** [ostinelli/gin#22]
Hi,ostinelli!
     I found that you set nginx directive lua_package_path in a func named gin_init, gin start execute with a param -e 'package.path, but as a result in my development-nginx.conf I got this 

```
lua_code_cache off;
    lua_package_path "./?.lua;$prefix/lib/?.lua;#{= LUA_PACKAGE_PATH };;";
```

 This made a mistake when i visit  http://localhost:7200/ like 

```
2015/08/11 17:38:07 [error] 79803#0: *3 lua entry thread aborted: runtime error: content_by_lua(development-nginx.conf:38):1: module 'gin.core.router' not found:
	no field package.preload['gin.core.router']
	no file './gin/core/router.lua'
	no file '/Users/zj-gin/sina//lib/gin/core/router.lua'
	no file '#{= LUA_PACKAGE_PATH }'
	no file '/usr/local/nginx_x/lualib/gin/core/router.lua'
	no file '/usr/local/nginx_x/lualib/gin/core/router/init.lua'
	no file './gin/core/router.lua'
	no file '/usr/local/nginx_x/luajit/share/luajit-2.1.0-alpha/gin/core/router.lua'
	no file '/usr/local/share/lua/5.1/gin/core/router.lua'
	no file '/usr/local/share/lua/5.1/gin/core/router/init.lua'
	no file '/usr/local/nginx_x/luajit/share/lua/5.1/gin/core/router.lua'
	no file '/usr/local/nginx_x/luajit/share/lua/5.1/gin/core/router/init.lua'
	no file '/usr/local/nginx_x/lualib/gin/core/router.so'
	no file './gin/core/router.so'
	no file '/usr/local/lib/lua/5.1/gin/core/router.so'
	no file '/usr/local/nginx_x/luajit/lib/lua/5.1/gin/core/router.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/nginx_x/lualib/gin.so'
	no file './gin.so'
	no file '/usr/local/lib/lua/5.1/gin.so'
	no file '/usr/local/nginx_x/luajit/lib/lua/5.1/gin.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
```

what's this? #{= LUA_PACKAGE_PATH }? a nginx conf syntactic sugar?

**Comment by idevz** [comment]
nobody answer this... I'm so sad :(

**Comment by leite** [comment]
don't be sad :)

I think it's related to #24 since you can see nginx (openresty) looking for gin paths in 5.1.

and #{= LUA_PACKAGE_PATH } is wrong according to [#lua_package_path](https://github.com/openresty/lua-nginx-module/#lua_package_path) should be $LUA_PACKAGE_PATH or ${LUA_PACKAGE_PATH}

@ostinelli what is that for [launcher.lua#L51](https://github.com/ostinelli/gin/blob/master/gin/cli/launcher.lua#L51) ?

**Comment by idevz** [comment]
thank you every much for your guys :)

**Comment by istr** [comment]
You might find [this commit](https://github.com/ostinelli/gin/commit/26eb8ee3927d8206ab91247607a4d06a40b1585c) useful to get your problem solved and integrate seamlessly into a custom build.

**Comment by idevz** [comment]
thanks @istr [[that commit]](https://github.com/ostinelli/gin/commit/26eb8ee3927d8206ab91247607a4d06a40b1585c) is another good way to solve this.

**Comment by ostinelli** [comment]
`LUA_PACKAGE_PATH` is a leftover, and has been removed. What discussed here should have been solved with #24. Closing.

Thank you for the inputs!
